src/cljs/ataru/virkailija/editor/handlers.cljs

```diff
@@ -519,3 +519,13 @@
     (let [path (conj (vec (current-form-content-path db path))
                      :belongs-to-hakukohteet)]
       (update-in db path (fnil (comp vec #(disj % oid) set) [])))))
+
+(reg-event-db
```

Comment:
Onkohan nää indirektiot lopulta tarpeellisia vs. vaan käyttää `(dispatch [:state-update #(assoc-in % [:editor :ui :all-folded) true])`?